bin/controller

```diff
@@ -18,17 +18,17 @@ Environment:
   BLUEPILL_LOGFILE - output log (default: log/bluepill.log)
   ROBOT_ENVIRONMENT - (default: development)
 '
-  exit -1
+  exit(-1)
 end
 
 ENV['ROBOT_ENVIRONMENT'] ||= 'development'
 ENV['BLUEPILL_BASE_DIR'] ||= File.expand_path('run/bluepill')
-ENV['BLUEPILL_LOGFILE']  ||= File.expand_path('log/bluepill.log')
```

Comment:
Here's one I actually don't like.  The former (aligned) version is clearly more readable.